wingbits/start.sh

```diff
@@ -38,18 +38,71 @@ fi
 echo "Settings verified, proceeding with startup."
 echo " "
 
-# Check if Wingbits is latest version
-
-local_version=$(cat /etc/wingbits/version)
+# Check if Wingbits is latest version and update if not
+# Determine OS
+case "$(uname -s)" in
+	Linux)
+		GOOS="linux"
+		;;
+	Darwin)
```

Comment:
Does it makes sense to "support" the darwin platform?

Reply:
Yes, I think so, it runs in a linux container, so it has to be linux, even if you run it on macOS.

Reply:
OK resolved this now :-)